components/ui/DynamicLink.tsx

```diff
@@ -10,12 +10,9 @@ interface DynamicLinkProps extends ExtraProps {
 
 export const DynamicLink = ({ children, href, ...props }: DynamicLinkProps) => {
   return (
-    // Because {children} contains <a> tags we need to wrap it in a div to pass className attributes in. If we inserted into the <Link> it doesnt get passed in. 
-    <div className="cursor-pointer">
-      {/* Work around for using legacyBehavior so that anchors in HTML in MDX files do not cause legacy errors */}
-      <Link href={href} {...props} legacyBehavior>
-        {children}
-      </Link>
-    </div>
+    <Link href={href} {...props} legacyBehavior>
+      <a className="cursor-pointer">{children}</a>
+    </Link>
```

Comment:
Can we fix up `legacyBehavior`

I know the code was already like that, but i think we should fix things like this as we go :)

Reply:
apoc, done